src/game.cpp

```diff
@@ -848,6 +851,59 @@ bool game::start_game()
     return true;
 }
 
+void game::place_starting_vehicle( const vproto_id &starting_vehicle )
```

Comment:
I wouldn't split these functions like that. It's nice to have a general purpose function which places vehicles on their default terrain unless specified otherwise. Having said that, I'd leave only `place_vehicle_nearby`, but use an empty default value for the last argument:
```c++
vehicle *place_vehicle_nearby( const vproto_id &id, const point &origin, int min_distance, int max_distance, const std::vector<std::string> &omt_search_types = {})`
```
Then you can check the value of `omt_search_types` for emptiness, and use the default vehicle-specific terrain types if it is empty.